packages/augur-ui/src/index.ejs

```diff
@@ -107,6 +107,12 @@
 
   <!-- Google Translate -->
   <script type="text/javascript">
+    if ((window.location.protocol === "http:" &&
+      (window.location.host !== "localhost" && window.location.host !== "127.0.0.1") &&
```

Comment:
```suggestion
      (window.location.hostname !== "localhost" && window.location.host !== "127.0.0.1") &&
```

See: https://stackoverflow.com/questions/6725890/location-host-vs-location-hostname-and-cross-browser-compatibility

Reply:
Ironically, while @justinbarry was correct in this feedback, if @bthaile's original code was used then local IPFS browsing wouldn't have broken (since it uses port 8080).  😄  (see #8808)